clang/include/clang/Basic/LangOptions.h

```diff
@@ -509,6 +519,10 @@ class LangOptions : public LangOptionsBase {
   /// records.
   std::string OptRecordFile;
 
+  std::string FPAccuracyVal;
+  using FPAccuracyFuncMapTy = std::map<std::string, std::string>;
```

Comment:
StringMap?

Reply:
hm! that doesn't seem to be working. Generating build errors. Doesn't it take StringRef and I am using std::string?

Reply:
It does, but I'm not certain what about this requires the allocations used by std::string, either.

Reply:
Left it as is for now. Will find out more about it.